neo/rawio/blackrockrawio.py

```diff
@@ -375,23 +391,45 @@ def _parse_header(self):
                 self._seg_t_stops.append(float(t_stop))
                 self._sigs_t_starts.append(float(t_start))
 
+            # Remap segments so they are a range again
+            for i in range(self._nb_segment):
+                try:
+                    self.nsx_data[i]
+                except KeyError:
+                    self.nsx_data = {key - 1 if key > i else key: value for (key, value) in
+                                     self.nsx_data.items()}
```

Comment:
With a quick overview I don't anderstand these lines. Maybe you could add more comments.
the **key -1 ** is not obvious.